packaging/ansible-runner-service-project/project/roles/ovirt-host-deploy-vdsm/tasks/configure.yml

```diff
@@ -11,6 +11,39 @@
 # grab local devices that should be blacklisted. This may expose
 # the local devices as FC storage in engine.
 
+- block:
+  - name: mask libvirt modular daemon units
+    systemd:
+      name: "{{ item }}"
+      state: stopped
+      enabled: no
+      masked: yes
+    with_items:
+      - virtqemud.service
+      - virtinterfaced.socket
+      - virtnetworkd.socket
+      - virtnodedevd.socket
+      - virtnwfilterd.socket
+      - virtsecretd.socket
+      - virtstoraged.socket
+      - virtproxyd.socket
+
+  - name: enable libvirtd monolithic daemon unit
+    systemd:
+      name: libvirtd
+      enabled: yes
+
+  - name: reboot el9 host
```

Comment:
Once we also remove the leftover `/var/run/libvirt/virtqemud-sock` we can skip the reboot altogether.